lib/index.js

```diff
@@ -24,6 +24,11 @@ class Organization {
     return this.client.get({ endpoint, query });
   }
 
+  manageableGroups(username) {
```

Comment:
I would call the param `userId`. Although we support usernames as legacy, user ids should be favored.